core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/BatchChainCustomizationInfo.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.internal.profiling.tracing.event.span;
+
+import static org.mule.runtime.core.privileged.profiling.tracing.ChildSpanCustomizationInfo.getDefaultChildSpanInfo;
+
+import org.mule.runtime.api.component.location.ComponentLocation;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.privileged.profiling.tracing.ChildSpanCustomizationInfo;
+
+/**
+ * Batch Chain Customization info. Includes batch-step-record, batch-aggregator and batch-on-complete spans.
+ *
+ * @since 4.5.0
+ */
+public class BatchChainCustomizationInfo extends AbstractDefaultAttributesResolvingSpanCustomizationInfo {
```

Comment:
Pass this to mule-ee. as this corresponds to batch. Maybe I was the one that set it here in the working branch. Maybe it can be generalized so that this is not related to batch. Possibly set a descriptive name and it can be reuse for other components. It simply is basing everything in the location and the name.